Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -2101,10 +2101,14 @@ inline void BuilderCustom::applyValueContainIntrinsicWidth(BuilderState& builder
     auto& list = downcast<CSSValueList>(value);
     ASSERT(list.length() == 2);
     ASSERT(downcast<CSSPrimitiveValue>(list.item(0))->valueID() == CSSValueAuto);
-    ASSERT(downcast<CSSPrimitiveValue>(list.item(1))->isLength());
-    style.setContainIntrinsicWidthType(ContainIntrinsicSizeType::AutoAndLength);
-    auto lengthValue = downcast<CSSPrimitiveValue>(list.item(1))->computeLength<Length>(builderState.cssToLengthConversionData().copyWithAdjustedZoom(1.0f));
-    style.setContainIntrinsicWidth(lengthValue);
+    ASSERT(downcast<CSSPrimitiveValue>(list.item(1))->isLength() || downcast<CSSPrimitiveValue>(list.item(1))->valueID() == CSSValueNone);
```

Comment:
I think this specific ASSERT is redundant as you do the same check in the if. You could move the length assert into else branch.

Reply:
Removed.